scripts/import/dbSNP_v2/update_new_sets.pl

```diff
@@ -0,0 +1,342 @@
+} else {
+  my $old_release = $release - 1;
+  my $old_dbname = $dbh->dbname =~ s/_${release}_/_${old_release}_/gr;
+  my $old_host = $dbh->host;
+  my $old_port = $dbh->port;
+  #Connect to DBI
+  $old_dbh = DBI->connect("DBI:mysql:database=${old_dbname};host=${old_host};port=${old_port}", "ensro", "");
+  debug($config, "Connected to the old database $old_dbname");
+}
+
+debug($config, "Selecting minimum and maximum variation feature");
+my $sql = qq{ SELECT MIN(variation_feature_id), MAX(variation_feature_id) FROM variation_feature};
+my $sth = $old_dbh->prepare($sql);
+my $sth = $dbh->prepare($sql);
+$sth->execute();
+my $vf = $sth->fetchall_arrayref();
+my $min_id = $vf->[0]->[0];
+my $chunk = 1000000;
+my $max_id = $vf->[0]->[1];
+
+debug($config, "Dumping the variation sets and the new variation feature into files");
+for my $tmp_num (map { $_ } $min_id/$chunk .. $max_id/$chunk) {
+  dump_old_sql_variation_sets($old_dbh, $tmp_num, $chunk, $max_id);
+}
+
+for my $tmp_num (map { $_ } $min_id/$chunk .. $max_id/$chunk) {
+  dump_new_variation_feature($dbh, $tmp_num, $chunk, $max_id);
+}
+
+debug($config, "Sorting the files");
+system(sort -u $TMP_FILE);
+system(sort -u $tmp_vset);
+
+debug($config, "Merging the files based on the new variation_id and the old variation set id");
+create_merged_file($tmp_vset, $TMP_FILE, $tmp_merged);
+
+debug($config, "Altering variation set variation table");
+temp_table($dbh);
+
+debug($config, "Loading new variation sets from merged file");
+load_all_variation_sets($dbh, $tmp_merged);
+
+debug($config, "Dumping new variation sets into a file to update the variation feature table");
+for my $tmp_num (map { $_ } $min_id/$chunk .. $max_id/$chunk) {
+  dump_new_variation_sets($dbh, $tmp_num, $chunk, $max_id);
+}
+
+debug($config, "Updating the variation feature table");
+update_variation_feature_table($dbh,$tmp_vs_file);
+
+
+
+debug($config, "Adding failed variation to variation set");
+$dbh->do(qq{ 
+  INSERT IGNORE INTO variation_set_variation (variation_id, variation_set_id)
+  SELECT DISTINCT variation_id, 1 
+  FROM failed_variation; 
+}) or die "Failed to add failed to variation_set_variation table";
+
+$dbh->do(qq{
+  ALTER TABLE variation_set_variation ENABLE keys;
+}) or die "Failed to alter variation_set_variation keys";
+
+
+sub temp_table { 
+  my $dbhvar = shift; 
+   
+  # creating a temp table and altering the table by disabling the keys 
+  my $alter_sql = $dbhvar->prepare(qq{ALTER TABLE variation_set_variation DISABLE keys});
+  #executing the sql 
+  $alter_sql->execute();
+
+  $alter_sql->finish();
+}
+
+sub create_merged_file {
+  # this would merge the files variation_feature.txt and variation_name_set.txt using the column they share in common which is the var_name and creates a new file 
+  # which is variation_name_set.txt containing the var_id and var_set_id which would be loaded to the new variation_set table
+  my $file = shift; 
+  my $second_file = shift; 
+  my $third_file = shift;
```

Comment:
It's difficult to read the code with these variable names. You could use the same file names everywhere.
This method is called in line 78 so here it could use the same file names as line 78.